src/version-2-0-0.md

```diff
@@ -0,0 +1,26 @@
+Date: 2016-03-08
+Title: Release 2.0.0
+Tags: news, release notes
+Slug: uniqush-push-2-0-0
+Category: news
+
+Please check out our [release notes] for version 2.0.0.
+
+We've improved the response format of most APIs. Instead of sending server logs
+back, they now return JSON response objects. This makes it easy to distinguish
+successes from failures, since, for example, a subscriber can have many
```

Comment:
Long sentence, split up

"For example, when sending push notifications to a subscriber with multiple delivery points, some may fail, but others may succeed. The JSON response object for a push contain easily parseable information on the status of each delivery point."